runtime/onert/core/include/backend/basic/BackendContextHelpers.h

```diff
@@ -24,6 +24,7 @@
 #include "util/logging.h"
 #include "backend/ITensorRegistry.h"
 #include "backend/BackendContext.h"
+#include "backend/train/TrainableBackendContext.h"
```

Comment:
@ragmani May I ask why you don't put these change under `backend/train/BackendContextHelpers.h`? You've put the changes under `train/` so far.

Reply:
OK I will introduce a new helper header. There will still be modifications of the `basic/BackendContextHelpers.h` to reduce code duplication.